Doppler.PushContact/Controllers/PushContactController.cs

```diff
@@ -54,6 +54,23 @@ public async Task<IActionResult> GetBy([FromQuery, Required] string domain, [Fro
             return Ok(pushContacts);
         }
 
+        [HttpPut]
+        [Route("push-contacts/update/visitor-guids")]
+        public async Task<IActionResult> UpdatePushContactVisitorGuid([FromBody] List<PushContactUpdateModel> pushContactUpdateModelList)
+        {
+            if (!pushContactUpdateModelList.Any() || pushContactUpdateModelList == null)
```

Comment:
Coortesía de @darianbenito, si pushContactUpdateModelList esto va a romper.

```suggestion
            if (pushContactUpdateModelList == null || !pushContactUpdateModelList.Any())
```

Reply:
Estaría bueno considerar estos escenarios en los tests.

Reply:
joya, ahi lo cambio y veo de agregar algun test.

Reply:
Al final no agregue test para esto porque los vamos a ir pasando de a uno, entonces no hay que hacer este check de la lista porque ya no hay lista.